README.md

```diff
@@ -34,6 +34,14 @@ For advanced docker-based usage, look at the full [Tracim Docker documentation](
 
 In order to contribute to the Tracim source code, please read [CONTRIBUTING.md](./CONTRIBUTING.md) file
 
+### Translation
+
+You can help Tracim with improvements or new translations, for this we use the [Weblate](https://hosted.weblate.org/projects/tracim/) platform.
```

Comment:
```suggestion
You can help us translate Tracim! If you see missing or wrong translations, head over to [Weblate](https://hosted.weblate.org/projects/tracim/).
```

Should we mention "contribute a new language"?

Reply:
```suggestion
You can help us translate Tracim! If you want to contribute with a new language or you see missing or wrong translations, head over to [Weblate](https://hosted.weblate.org/projects/tracim/).
```
Like that @raphj?

Reply:
Yes, maybe contribute with -> contribute

(to contribute something)